x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackClientPlugin.java

```diff
@@ -343,7 +343,7 @@ static Settings additionalSettings(final Settings settings, final boolean enable
                 PutLifecycleAction.INSTANCE,
                 ExplainLifecycleAction.INSTANCE,
                 SetIndexLifecyclePolicyAction.INSTANCE,
-                RemovePolicyForIndexAction.INSTANCE,
+                RemoveIndexLifecyclePolicyAction.INSTANCE,
```

Comment:
Im going to assume all of these changes are approved by someone else :)

Reply:
Name changes match the changes to SetIndexLifecyclePolicyAction.

Reply:
Done.